requirements/app.in

```diff
@@ -48,6 +48,7 @@ Flask-Security-Too>=4.0
 Flask-Classful
 Flask-Marshmallow
 Flask-Cors
+sentry-sdk[flask]
```

Comment:
Can you comment on this notation?

Reply:
This is the common syntax for libraries that also install to load certain extensions. In this case, Flask is the extension. As it is rather standard, I'd like to not add a comment.